cmd/snap/main.go

```diff
@@ -232,11 +232,10 @@ func run() error {
 		if e, ok := err.(*client.Error); ok && e.Kind == client.ErrorKindLoginRequired {
 			u, _ := user.Current()
 			if u != nil && u.Username == "root" {
-				return fmt.Errorf(`%s (see "snap login --help")`, e.Message)
```

Comment:
i see no removal :-) but the diff is harder to read than it could. There are two things: the additon of `i18n.G` to both `fmt.Errorrf`s, and the outdent of the second `fmt.Errorf` prompted by golint